### PR TITLE
Add a script to export logs in CI

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Build/Testing/Packaging Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
System tables from CI checks are exported to ClickHouse Cloud.

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit 8b7f8ad910ab76af78eaca1a9c6e4b5b2d830fd4 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/53086/8b7f8ad910ab76af78eaca1a9c6e4b5b2d830fd4/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: docker/test/stateless/run.sh
Comment (Felixoid) [inline docker/test/stateless/run.sh:126]: Please, don't merge it. It will be impossible to debug with GH actions -> python -> docker -> mounter repo

We should not use scripts from the repo in docker containers, it's no way practice that should be prevented

**Review by Felixoid — CHANGES_REQUESTED** [review]

File: docker/test/stateless/run.sh
Comment (Felixoid) [inline docker/test/stateless/run.sh:113]: there should be a condition to break the loop. Like `for i in {1..100}`. And if nothing starts, the whole script should fail

File: programs/server/config.xml
Comment (alexey-milovidov) [reply programs/server/config.xml:1042]: No, this is the default.

File: docker/test/stateless/setup_export_logs.sh
Comment (Felixoid) [inline docker/test/stateless/setup_export_logs.sh:1]: It shouldn't work. Does it work?

Docker usually rejects copying anything out of the context directory, which is `
docker/test/stateless` in this case

Comment (Felixoid) [reply docker/test/stateless/setup_export_logs.sh:1]: Ah, ok. It works now but will break after the #52865 will be merged. The `setup.sh` should be in the docker directory and linked to utils.

File: tests/ci/functional_test_check.py
Comment (Felixoid) [inline tests/ci/functional_test_check.py:393]: What `subprocess.check_call` will cause in case both of the ENV are empty?

File: tests/ci/functional_test_check.py
Comment (Felixoid) [inline tests/ci/functional_test_check.py:394]: It looks safer

```suggestion
    ci_logs_host = os.getenv("CLICKHOUSE_CI_LOGS_HOST", "CLICKHOUSE_CI_LOGS_HOST")
    ci_logs_password = os.getenv("CLICKHOUSE_CI_LOGS_PASSWORD", "CLICKHOUSE_CI_LOGS_PASSWORD")
```

File: docker/test/stateless/setup_export_logs.sh
Comment (alexey-milovidov) [reply docker/test/stateless/setup_export_logs.sh:1]: Ok. I don't need the `utils/export-logs-in-ci/setup.sh` script, but I want it in almost every Docker image: stateless, stateful, stress, fuzzer... How to do it?

Comment (Felixoid) [reply docker/test/stateless/setup_export_logs.sh:1]: If looks like you need to put it into base https://github.com/ClickHouse/ClickHouse/blob/master/docker/images.json#L120

Or, if in fasttests too, than to util https://github.com/ClickHouse/ClickHouse/blob/master/docker/images.json#L120

Comment (alexey-milovidov) [reply docker/test/stateless/setup_export_logs.sh:1]: I will put it to `base`

**Pull request merged**
